### ISSUE TITLE
Mobile application 

### ISSUE THREAD

**Issue by davidnajar** [GOATS2K/Coral#119]
Hey, I just discovered your project today while looking for information about solving an issue with plugin approach that I'm facing in a project I'm working on. Great work! 

Did not yet tried to install nor play with it, but just looking at Plex or jellyfin, they both have android and iOS applications. Do you plan to include it in coral? Or at least, open for someone else to collaborate?

**Comment by davidnajar** [comment]
Ok, I saw #111 now,  is the plan to migrate everything to RN, including the web itself?

**Comment by GOATS2K** [comment]
That's correct, I'm looking into porting the entire UI to React Native using [tamagui](https://tamagui.dev). I've been daily driving Coral for the past few weeks and it's slowly getting there, but there's still a lot of stuff to improve on (as you can tell from the issues).


I hope you managed to fix your plugin solution! I've changed Coral's plugin-system a little since its original inception, but the plugin's core works the exact same. I'm going to try out a gRPC based approach in the future so we can take advantage of Native AOT once it's more stable. :)
